http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeService.scala

```diff
@@ -34,6 +34,14 @@ trait RuntimeService[F[_]] {
     as: Ask[F, AppContext]
   ): F[Vector[ListRuntimeResponse2]]
 
+  def listRuntimes(userInfo: UserInfo,
```

Comment:
Why create a new listRuntime function instead of modifying the previous one?

Reply:
Bleh, okay, it's not you, it's scala, I wish there was a clearer way to set this field as optional :(

Reply:
hmm could you not just add the following to the original function?
`excludeStatuses: List[RuntimeStatus] = List(RuntimeStatus.Deleted)`

that would set a default of excluding deleted runtimes